CHANGELOG.md

```diff
@@ -28,10 +35,9 @@
 * Fixed an issue where running in a private gitlab repo caused a warning message to be shown multiple times.
 * Added a new validation to the **validate** command to verify that markdown and python files do not contain words related to copyright section.
 * Fixed an issue where **lint** crashed when provided an input file path (expecting a directory).
-* Added a new validation to **validate**, making sure all inputs of a playbook are used.
-* Added a new validation to **validate**, making sure all inputs used in a playbook declared in the input section.
```

Comment:
were these misplaced here?

Reply:
No
It is only merged in the last week to the master.